training/unit-testing/go-lang/README.md

```diff
@@ -49,75 +31,81 @@ In your terminal:
-- [Dependency Injection within Mockups](https://github.com/sohamkamani/go-dependency-injection-example)
+This dependency injection example explains the need for dependency injection, and how it can be implemented in Go: [Dependency Injection within Mockups](https://github.com/sohamkamani/go-dependency-injection-example)
 
 #### Test Driven Development
-A useful approach, particularly for potentially difficult to test code, is Test Driven Development, which involves writing the test at the same time or before the code you need to test. So effectively designing easy to test code from the start.
-- [Golang Test Driven Development walk through](https://itnext.io/how-to-tdd-a-console-application-to-achieve-100-coverage-starting-from-main-test-go-934a617b080f) 
+A useful approach, particularly for potentially difficult to test code, is Test Driven Development, which involves writing the test at the same time, or before the code you need to test. So effectively designing easy to test code from the start.
 
-### Recommended Go Tutorials
+[Here is a Go Test Driven Development walk through](https://itnext.io/how-to-tdd-a-console-application-to-achieve-100-coverage-starting-from-main-test-go-934a617b080f)
+
+#### Recommended Go Tutorials
 - [GoConvey Tutorial 1](https://www.smartystreets.com/blog/2015/02/go-testing-part-1-vanillla/)
 - [GoConvey Tutorial 2](https://www.smartystreets.com/blog/2015/02/go-testing-part-2-running-tests/)
 - [GoConvey Tutorial 3](https://www.smartystreets.com/blog/2015/05/go-testing-part-3-convey-behavior/)
 
-### Code Coverage
-When we test certain paths of execution are tested, but often not all.  Knowing which paths are tested is useful, but often the same paths need to be tested multiple time, so 100% coverage doesn't mean 100% tested.
+## Code Coverage
+When we test, certain paths of execution are run, but often not all.  Code Coverage tells us which code was executed during a test.  Sometimes the same paths need to be tested multiple times, but its good to limit this.  It's also important to note that 100% code coverage doesn't mean 100% tested.
+
+Here are two useful links to read:
 - [Excellent Golang code coverage walk-through](https://blog.golang.org/cover)
 - [Useful time saving suggestion on stackoverflow](https://stackoverflow.com/a/27284510)
 
-### Assessing How Fast Your Code Runs
-Systems often need to run efficiently, such as a response from a website.  Go incorporates a benchmarking routines, which will time how long the code will take to run.  It clever though as it repeats the tests until it feel it's got an accurate time.  
-- [Downloadable benchmarking example](benchmarking/benchmark_test.go)
+[dp-identify-api]() is a great ONS example of a nicely designed, loosely coupled repo.  Its design also limits the code which is re-run as part of the tests, therefore saving resources.
```

Comment:
Missing link to repository - https://github.com/ONSdigital/dp-identity-api

Reply:
Whoops - thanks!